src/ccunningham_calc.java

```diff
@@ -0,0 +1,101 @@
+
+import java.util.Scanner;
+
+class ccunningham_calc {
+	
+	public static void main(String[] args){
```

Comment:
You have placed all of your logic in the main statement. First, it is not a good idea to do this as it defeats the point of object-oriented programming (that is one of the aspects of working in static space). Second, you typically don't want to call into your method into a main statement. Every class in java may have a main statement, but your program should really only call into a single one (in this case the one in calc.java). The others are for internal/debug purposes. Finally, running the code in a large code-block like this is error prone and difficult to maintain. It would have been much better to break it into several functions.